lib/legacy/class.html

```diff
@@ -14,18 +14,38 @@
 
     'use strict';
 
-    let metaProps = {
+    const metaProps = {
       attached: true,
       detached: true,
       ready: true,
       created: true,
       beforeRegister: true,
       registered: true,
       attributeChanged: true,
-      // meta objects
-      behaviors: true
     };
 
+    const noBehaviorCopyProps = Object.assign({
+      behaviors: true
+    }, metaProps);
+
+    const memoizedProps = Object.assign({
```

Comment:
Naming of these is confusing.  Consider "memoized" --> "filtered"

Reply:
Done.